#goal definition
int32 num_agent
---
#result definition
bool sucess
---
#feedback
